data_scripts.csh

```diff
@@ -42,13 +45,11 @@ else if ($CONTINUE_RUN == TRUE) then
 
    # If the assimilation ended on the first day of the month
    # then the month assimilated is the previous.
-   if ($ATM_DATE[3] == '01' ) then
-      if ($data_month == 1) then
-         @ data_month = 12
-         @ data_year  = $data_year - 1
-      else
-         @ data_month = $data_month - 1
-      endif
+   if ($data_month == 1) then
+      @ data_month = 12
+      @ data_year  = $data_year - 1
+   else
+      @ data_month = $data_month - 1
```

Comment:
This code no longer matches the logic of the comment ... not sure how 'data_month' and 'data_year' are used in all cases. we need to follow this through or rename variables to be more specific about intended use.

Reply:
True enough.  I've fixed the comment and will push it after we resolve
other issues in the PR.